### DISCUSSION TITLE
Getting error on manual run. Not sure what's wrong.

### DISCUSSION THREAD

**Discussion by bonelifer** [upptime/upptime#961] [Q&A] [answered]
Not sure what I'm doing wrong, copied from the instructions page:

https://github.com/phpbbmodders/status/blob/master/.github/workflows/.upptimerc.yml

```

Annotations
1 error

No event triggers defined in `on`
```

**Comment by AnandChowdhary** [comment, marked as answer]
`.upptimerc.yml` should not be in `.github/workflows` but in your project root. Then you can run "Setup CI" manually.
